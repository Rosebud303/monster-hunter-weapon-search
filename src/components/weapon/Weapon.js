import React from 'react'

export const Weapon = () => {
  return (
    <div>
      hi
    </div>
  )
}

export default Weapon;
