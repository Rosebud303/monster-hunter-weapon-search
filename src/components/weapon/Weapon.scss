.weapon-card{
  font-size: 16px;
  font-weight: 900;
  font-family: monospace;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  background: black;
  color: white;
  text-decoration: none;
  margin-bottom: 3%;
  width: 22%;
  border-radius: 10px;
  border: 2px solid #33334a;
  &:hover {
    transition: .3s;
    box-shadow: inset 0 0 0 5px #53a7ea;
  }
}

.weapon-image{
  width: 90%;
}