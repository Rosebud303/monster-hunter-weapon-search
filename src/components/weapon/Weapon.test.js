import React from 'react';
import { shallow } from 'enzyme';
import Weapon from './Weapon'


describe('Weapon', () => {
  let wrapper; 
  let mockId = 1
  let mockPicture = 'picture.png'
  let mockName = 'Sword'
  beforeEach(() => {
    wrapper = shallow (
      <Weapon id={mockId}
              name={mockName}
              picture={mockPicture}/>
    )
  });

  it('should match snap shot', () => {
    expect(wrapper).toMatchSnapshot()
  });

});