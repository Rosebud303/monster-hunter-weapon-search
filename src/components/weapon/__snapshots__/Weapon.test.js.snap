// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weapon should match snap shot 1`] = `
<Link
  className="weapon-card"
  id={1}
  to="/weapons/1"
>
  <p
    id={1}
  >
    Sword
  </p>
  <img
    className="weapon-image"
    id={1}
    src="picture.png"
  />
</Link>
`;
